index.html

```diff
@@ -0,0 +1,116 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Personal Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet">
+</head>
+
+<body>
+    <header class="header-nav">
+        <h3>SheMer</h3>
```

Comment:
For HTML semantic purposes, I would kindly suggest you use h1 tag for your header title as opposed to h3. You want your logo to stand out above the other text on your document.

Reply:
- [ ] [OPTIONAL] Good use of inline comment, @tsheporamantso! However, you want to be sure of the quality of your suggestions when you give a code review ♻️  so that the student doesn't get confused. When you are not sure of your suggestion, it's a good thing to make findings or ask a senior colleague. Usually, and for semantic purposes, logos like this on a page is best wrapped in a `a` tag so that it be a reference to the homepage from any part of the website once clicked. This will also improve accessibility 😉

Reply:
Highly noted, Thank you.